packages/core/bookingLimits/getBusyTimesFromLimits.ts

```diff
@@ -123,6 +129,8 @@ const _getBusyTimesFromBookingLimits = async (params: {
             teamId,
             user,
             rescheduleUid,
+            includeManagedEvents,
+            timeZone,
```

Comment:
not passing timeZone here caused this error with yearly booking limits: 
<img width="435" alt="Screenshot 2024-10-04 at 9 06 34 PM" src="https://github.com/user-attachments/assets/4239bcfb-d36c-40d0-8a24-972fbaee9cb7">